packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -337,6 +338,37 @@ class FlutterPlugin implements Plugin<Project> {
         }
     }
 
+    // Add the dependencies on other plugin projects to the plugin projects.
```

Comment:
// Add the dependencies on other plugin projects to the plugin project.

The plugin project is singular right?